### ISSUE TITLE
[Quest - Repeatable] Encrypted Twilight Text

### ISSUE THREAD

**Issue by alexuktc** [dalaranwow/dalaran-wow#4566]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After completing a chain of quests for Cenarion Circle in Silithus you get a repeatable quest called Encrypted Twilight Texts. Each turn in gives 250 reputation. 
**Expected behaviour**: Tell us what should happen instead
Each turn-in should give 500 rep, effectively halving the grind for this faction. This was changed in 2.3

**Steps to reproduce the problem**:

1. Get quest items
2. Turn-in quest
3. Notice the lower rep gain.

**Include proofs for this behaviour**
http://www.wowhead.com/item=20404/encrypted-twilight-text#comments - Quest items
http://www.wowhead.com/quest=8319/encrypted-twilight-texts - Quest itself

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
item=20404 - Item
quest=8319 -Quest

No screenshot as I already grinded my ass off for this, but I think it should be corrected for people doing this in the future.

**Comment by Teridan-dal-wow** [comment]
Confirmed. @alexuktc thanks for the report!
Fix:
``` sql
UPDATE `quest_template` SET `RewardFactionValue1` = 7  WHERE `ID` = 8319; 
```
How it looks now:
![image](https://cloud.githubusercontent.com/assets/6797611/23986993/c63c482e-0a2f-11e7-830f-3c7cb0994959.png)

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/2335, already fixed on PTR.

**Comment by nigolaj** [comment]
The problem is back, the quest gives only 250 reputation.
